### ISSUE TITLE
Module bardapi not recognised anymore

### ISSUE THREAD

**Issue by Alexo47** [acheong08/Bard#80]
I have been using this script that worked perfectly:

!pip install bardapi==0.1.23a
from bardapi import Bard
myToken = 'xxxxxxxx'
bard = Bard(token=myToken)

Now I get systematically an error 'ModuleNotFoundError: No module named 'bardapi'
 at line ' from bardapi import Bard

Something changed?

Thanks for the attention, looking forward for feedback.

Cordially,
Alexandre

**Comment by RobinHood257** [comment]
Yes, it seems there have been many changes. attributes like " bard.ask() " or " bard.generate() " are also do not work any more!

I used this code:
"""
import bard

response = bard.ask(input("Say Something! : "))

print(response)
"""

**Comment by Alexo47** [comment]
Thanks!
My understanding is that my issue is related to management of packages in my development environment which was Jupiter Notebooks. I was not able to restore the situation that worked without problems.

I changed the developing environment to Pycharm Community and was able to install bardapi package and got no more problems with the same python script I was using  in jupyter notebooks.

All these issues of establishing the right environment are still very confusing for me.

Thanks for the attention and feedback are welcome.

Cordially,
Alexandre
